core/src/main/java/uk/ac/bbsrc/tgac/miso/core/event/manager/ProjectAlertManager.java

```diff
@@ -296,4 +296,49 @@ public void updateGroupWatcher(Long userId) throws IOException {
       }
     }
   }
+
+  /**
+   * Creates a minimal copy of the project to be used for change tracking in the alerting system. Only relevant fields are copied
+   * 
+   * @param project
+   *          the project to copy
+   * @return the copy
+   */
+  private Project partialCopy(Project project) {
+    Project clone = new ProjectImpl();
+    clone.setId(project.getId());
+    clone.setAlias(project.getAlias());
+    clone.setName(project.getName());
+    clone.setProgress(project.getProgress());
+    Set<ProjectOverview> projectOverviews = Sets.newHashSet();
+    for (ProjectOverview projectOverview : project.getOverviews()) {
+      projectOverviews.add(partialCopy(projectOverview));
+    }
+    clone.setOverviews(projectOverviews);
+
+    for (User u : project.getWatchers()) {
+      clone.addWatcher(u);
+    }
+    return clone;
+  }
+
+  /**
+   * Creates a minimal copy of the ProjectOverview to be used for change tracking in the alerting system. Only relevant fields are copied
+   * 
+   * @param projectOverview
+   *          The ProjectOverview to copy
+   * @return the copy
+   */
+  private ProjectOverview partialCopy(ProjectOverview projectOverview) {
+    ProjectOverview clone = new ProjectOverview();
+    clone.setId(projectOverview.getId());
+    clone.setProject(projectOverview.getProject());
```

Comment:
Can the partialCopied project be used here instead? It shouldn't affect functionality, but it would prevent the full project (including samples, libraries, runs, etc.) from being kept in memory unnecessarily